viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1089,10 +1114,12 @@ export class Cognite3DViewer {
       x: (offsetX / this.renderer.domElement.clientWidth) * 2 - 1,
       y: (offsetY / this.renderer.domElement.clientHeight) * -2 + 1
     };
+
     const input: IntersectInput = {
       normalizedCoords,
       camera: this.camera,
       renderer: this.renderer,
+      clippingPlanes: this.getSlicingPlanes(),
```

Comment:
Renamed to getClippingPlanes(). Will set up a PR for renaming setSlicingPlanes to setClippingPlanes, but since this is already released in 2.0 I can't just remove it (will mark as deprecated)

Reply:
See https://github.com/cognitedata/reveal/pull/1618